client/client.go

```diff
@@ -180,6 +180,9 @@ func (c *Client) ExecuteRequestURIParams(ctx context.Context, inputs RequestInpu
 
 	dateHeader := time.Now().UTC().Format(time.RFC1123)
 	req.Header.Set("date", dateHeader)
+	req.Header.Set("Accept", "application/json")
```

Comment:
I suggest normalizing the capitalization for headers so that all headers use the same form.